pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -75,11 +75,6 @@ func addControllerRoleToSA(saNamespace, saName string, role rbac.ClusterRole) {
 		}
 	}
 
-	if role.Annotations == nil {
```

Comment:
I moved all of this logic to a single place in `pkg/cmd/server/bootstrappolicy/policy.go` / `GetBootstrapClusterRoles`